packages/govern-server/src/api-graphql/resolvers.ts

```diff
@@ -0,0 +1,28 @@
+import { IResolvers } from 'apollo-server'
+import GovernData from '../core/data'
+
+export default function resolvers(govern: GovernData): IResolvers {
+  return {
+    Query: {
+      async dao(_, args) {
+        return govern.dao(args.address)
```

Comment:
Noice! The idea of the "API abstraction layer" looks really clean in his first draft/version 💪